cypress/e2e/dashboard.cy.js

```diff
@@ -68,243 +62,248 @@ describe('Validate dashboard page', () => {
     securityPo.breadcrumbs().should('not.exist')
   })
 
-  it.skip('Validate the EI card "Application" section in EN and FR', () => {
-    profilePo.ValidateCardTaskListAndSection(
+  it('Validate the EI card "Application" section in EN and FR', () => {
+    dashboardPo.ValidateCardTaskListAndSection(
       'Employment Insurance',
       'Applications',
       '2'
     )
     dashboardPo.FrenchButton().click()
-    profilePo.ValidateCardTaskListAndSection(
+    dashboardPo.ValidateCardTaskListAndSection(
       'Assurance-emploi',
       'Demandes de prestations',
       '2'
     )
   })
 
-  it.skip('Validate the EI card "Payments and claims" section', () => {
-    profilePo.ValidateCardTaskListAndSection(
+  it('Validate the EI card "Payments and claims" section', () => {
+    dashboardPo.ValidateCardTaskListAndSection(
       'Employment Insurance',
       'Payments and claims',
       '3'
     )
     dashboardPo.FrenchButton().click()
-    profilePo.ValidateCardTaskListAndSection(
+    dashboardPo.ValidateCardTaskListAndSection(
       'Assurance-emploi',
       'Paiements et demandes',
       '3'
     )
   })
 
-  it.skip('Validate the EI card "Taxes" section', () => {
-    profilePo.ValidateCardTaskListAndSection(
+  it('Validate the EI card "Taxes" section', () => {
+    dashboardPo.ValidateCardTaskListAndSection(
       'Employment Insurance',
       'Taxes',
       '2'
     )
     dashboardPo.FrenchButton().click()
-    profilePo.ValidateCardTaskListAndSection('Assurance-emploi', 'Impôts', '2')
+    dashboardPo.ValidateCardTaskListAndSection(
+      'Assurance-emploi',
+      'Impôts',
+      '2'
+    )
   })
 
-  it.skip('Validate the EI card "Reports and documents" section', () => {
-    profilePo.ValidateCardTaskListAndSection(
+  it('Validate the EI card "Reports and documents" section', () => {
+    dashboardPo.ValidateCardTaskListAndSection(
       'Employment Insurance',
       'Reports and documents',
       '7'
     )
     dashboardPo.FrenchButton().click()
-    profilePo.ValidateCardTaskListAndSection(
+    dashboardPo.ValidateCardTaskListAndSection(
       'Assurance-emploi',
       'Rapports et documents',
       '7'
     )
   })
 
-  it.skip('Validate the EI card "Reports and documents" section', () => {
-    profilePo.ValidateCardTaskListAndSection(
+  it('Validate the EI card "Personal information" section', () => {
+    dashboardPo.ValidateCardTaskListAndSection(
```

Comment:
Would it be possible to combine these tests together so they check the sections all at once so that it runs faster?

Reply:
Yes, going to create a create a PR soon for code optimization